testdata/baselines/reference/submodule/compiler/declarationEmitForGlobalishSpecifierSymlink.types.diff

```diff
@@ -0,0 +1,9 @@
+--- old.declarationEmitForGlobalishSpecifierSymlink.types
++++ new.declarationEmitForGlobalishSpecifierSymlink.types
+@@= skipped -39, +39 lines =@@
+
+ === /p2/index.d.ts ===
+ export const a: import("typescript-fsa").A;
+->a : import("/p2/node_modules/typescript-fsa/index").A
++>a : import("/p1/node_modules/typescript-fsa/index").A
```

Comment:
This is technically fixed by #1604 in that that PR would instead say `import("typescript-fsa")` instead.